module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -68,8 +68,8 @@ namespace module::localisation {
                                             const message::support::FieldDescription& field,
                                             const Eigen::Matrix<Scalar, 4, 4>& Hcw) const {
 
-            const Eigen::Matrix<Scalar, 4, 1> rBWw(state[PX], state[PY], field.ball_radius, 1.0);
-            const Eigen::Matrix<Scalar, 3, 1> rBCc_cart((Hcw * rBWw).template head<3>());
+            const Eigen::Matrix<Scalar, 3, 1> rBWw(state[PX], state[PY], field.ball_radius);
```

Comment:
Should this be a vector3 to be consistent?

Reply:
Instead of `Matrix<Scalar, 3, 1>`? If yes, you can't do a templated `Scalar` type with the vector typedefs